tools/tvk-oneclick/tvk-oneclick.sh

```diff
@@ -301,22 +314,29 @@ EOF
 #This module is use to install license
 install_license() {
   tvk_ns=$1
+  ret=$(kubectl get license -n "$tvk_ns" 2>> >(logit) | grep -q Active)
+  ret_code=$?
+  ret=$(kubectl get license -A 2>> >(logit) | grep -q Active)
+  ret_code_A=$?
+  if [ "$ret_code" -eq 0 ] || [ "$ret_code_A" -eq 0 ]; then
+    echo "License is already installed and is in active state"
```

Comment:
If License is already present and not in `active` state then you'll have to update the same license with new key as only  one License creation is allowed.

Reply:
Done modifications to handle this situation